parser_test.go

```diff
@@ -5,8 +5,86 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stripe/veneur/samplers"
+	"github.com/stripe/veneur/ssf"
 )
 
+func freshSSFMetric() *ssf.SSFSample {
```

Comment:
Great point. We destroy timestamps since we're gonna mush them anyway. I just realized we have no "ignore old samples" code in here, but I guess that's ok since we destroy them anyway.

In other words, we can work without a timestamp!